### PR TITLE
style: add links & bit formatting

### PR DESCRIPTION
**Pull request by vermasavita**
Bit formatting and remove commented code

### CODE REVIEW

**Review by Abhijeet43 — APPROVED** [review]
Looks good!!

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.ipt {
+    display: flex;
+    flex-direction: column;
+}
+
+.ipt-box {
+    color: var(--text-color);
+    font-size: 1.2rem;
+    line-height: 20px;
+    margin: 0 20px 10px;
```
Comment (mohammed-sas) [inline components/input/input.css]: since you are making your page responsive, i would suggest you to use rems here

**Review by mohammed-sas — APPROVED** [review]
other than suggested, your code is good to merge

File: components/input/input.css (hunk 1 above)
Comment (vermasavita) [reply components/input/input.css]: okay

**Pull request merged**